### DISCUSSION TITLE
Error in Academy Working with documents?

### DISCUSSION THREAD

**Discussion by SGerry** [pimcore/pimcore#9677] [Q&A / Support]
Topic of Question 5: What concepts can be used to add or reorder multiple elements in a template without editing the template file?

I am pretty sure, that Area bricks are a concept of adding or reorder multiple elements without editing the template file. But since this wasn't the correct answer, i tried ALL possible combinations including Area bricks. Now i tried 11 out of 16 combinations, but those don't seem to make a lot of sense.
I also tried "none" because you have to edit the tempalte file at least one time (at the beginning).
Also Blocks in combination with Area Bricks.

The only combinations i didn't tried out are [Sched. Blocks] , [Renderlet, Sched. Blocks] , [Renderlet, Blocks] and [Renderlet, Blocks, Sched. Blocks].

Can this really be? I am very confused about it

**Comment by fashxp** [comment]
Hi, 
in documents area bricks and blocks can be used to add or reorder elements without editing the template file. Renderlets just render the content that was dropped to them, and scheduled block render content depending on a timeline. 
So the correct answer to that question would be area bricks and blocks. 

BR
Christian

**Comment by SGerry** [comment]
Hi Christian,
This was a try on the first day, i made a photograph before sending. It was
not accepted, so i wondered why all possible combinations with area bricks
didn't work.
Mabe there was a display bug in the formular on my browser. I try again
next monday, since this was the very only error i had, i should be
successfull then ready for the senior :)

Thank you, best regards
Gerald

Am Di., 6. Juli 2021 um 08:28 Uhr schrieb Christian Fasching <
***@***.***>:

> Hi,
> in documents area bricks and blocks can be used to add or reorder elements
> without editing the template file. Renderlets just render the content that
> was dropped to them, and scheduled block render content depending on a
> timeline.
> So the correct answer to that question would be area bricks and blocks.
>
> BR
> Christian
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/pimcore/pimcore/discussions/9677#discussioncomment-968515>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFVDTGHCK4KAP5VYMSSWDLTTWKPBNANCNFSM472SU7WQ>
> .
>


-- 
Dipl.-Ing. (FH) Gerald Sittly
Senior Web-Entwickler

Travix Media GmbH
Alte Poststraße 4
D - 70173 Stuttgart


Telefon: +49 711 341 816 33
Telefax: +49 711 341 816 24

E-Mail: ***@***.***
Internet:  <http://www.travix-media.de/> http://www.travix-media.de

-- 

Registergericht: Stuttgart || Registernummer: HR 746913
Geschäftsführer: 
Jörg Hartenstein und Daniel Sommerfeld
USt-IdNr.: DE292345372
